### ISSUE TITLE
Bug: Duplicate records in Addresses, Employers and Organization_Memberships

### ISSUE THREAD

**Issue by PrabhakarThummalaDOL** [18F/dol-whd-14c#825]
Sometimes application inserts duplicate rows in Addresses, Employers and Organization_Memberships tables. 
- [x] Troubleshoot and fix.

@NathayaMDOL1  
On the employer registration page:
- [x] Restrict the user to click on the 'Register Employer' button only once.

**Comment by NathayaMDOL1** [comment]
@NathayaMDOL1. I'm here

**Comment by PrabhakarThummalaDOL** [comment]
The problem in this story has been addressed by the mouse stutter fix. A new story has been created with a related problem See #923.

**Comment by NathayaMDOL1** [comment]
Related to #920
